File: JobManagement.API/JobManagement.API/Data/Models/Jobs.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace JobManagement.API.Data.Models
{
    [Table("JOBS")]
    public class Jobs
    {
        [Column("ID"), Key]
        public int Id { get; set; }

        [Column("NAME")]
        public string Name { get; set; }

        [Column("DESCRIPTION")]
        public string Description { get; set; }

        [Column("USER_ID")]
        public int User_Id { get; set; }

        [Column("PHOTO")]
        public string Photo { get; set; }

        [Column("STATE")]
        public int State { get; set; }
    }
}

File: job/jobs.php
<?php

    class Job{

        public $Id;
        public $Name;
        public $Description;
        public $User_Id;
        public $Photo;
        public $State;

        public function initializeData($id,$name,$description,$user_id,$state)
        {
            $this->Id = $id;
            $this->Name = $name;
            $this->Description = $description;
            $this->User_Id = $user_id;
            $this->State = $state;                                         
        }

        public function Set($element){
            foreach($element as $key => $value){
                $this->{$key} = $value;
            }
        }

    }

?>